source/carla_host.py

```diff
@@ -2281,6 +2282,13 @@ def timerEvent(self, event):
     # close event
 
     def closeEvent(self, event):
+        if not self.fCustomStopAction == 1 and self.fSavedSettings[CARLA_KEY_MAIN_CONFIRM_EXIT]:
+            ask = QMessageBox.question(self, self.tr("Quit"), self.tr("Are you sure you want to quit Carla?"), QMessageBox.Yes|QMessageBox.No)
```

Comment:
small nitpick. Line is too long here.
often I align them with the start of the first `self.tr()`, so the 2nd one looks similar.
not all code in carla is good space-wise, but I dont want to add more to it..